packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -239,9 +246,8 @@ export interface BrowserPermissionDeniedStyles extends BaseCustomStyles {
     troubleshootingLink?: ILinkStyles;
 }
 
-// @public
-export interface CallAdapter extends AdapterState<CallAdapterState>, Disposable, CallAdapterCallManagement, CallAdapterDeviceManagement, CallAdapterSubscribers {
-}
+// @public (undocumented)
+export type CallAdapter = Omit<CallAdapterCommon, keyof ACSCallManagement> & ACSCallManagement;
```

Comment:
This Omit can be replaced by 'interface' method as we discussed @prprabhu-ms, will be working on it soon